src/vapid-helper.js

```diff
@@ -90,6 +96,48 @@ function validatePrivateKey(privateKey) {
 }
 
 /**
+ * Calculates the total expiration by adding
+ * value of Date.now() with the seconds passed
+ *
+ * @param {Number} numSeconds Number of seconds to calculate expiration
+ * @return {Number} expiration in seconds
+ */
+function calculateExpiration(numSeconds) {
+  return Math.floor(Date.now() / 1000) + numSeconds;
```

Comment:
@marco-c Was making this change, just for the record the above code has a typo. It should be `new Date()` instead of `Date.now()` and since we don't seem to be changing the reference to date. I think we can assume that we can declare it as a const.